src/main/java/org/embulk/input/config/ConfigInputPlugin.java

```diff
@@ -181,8 +182,10 @@ public void jsonColumn(Column column) {
                                 pageBuilder.setNull(column);
                             } else {
                                 try {
-                                    pageBuilder.setJson(column, jsonParser.parse(value.toString()));
-                                } catch (JsonParseException ex) {
+                                    pageBuilder.setJson(column, jsonParserBuilder.build(value.toString()).readJsonValue());
+                                } catch (final JsonParseException ex) {
+                                    throw new DataException(ex);
```

Comment:
Nit: Do we need `TODO` comment like `util-csv`?
https://github.com/embulk/embulk-parser-csv/pull/6/files#diff-cf388e19f294b56513c5fdab2f16a64ef08dff3d22365a969a84c9652a3cfdb1R425

Reply:
Ah, for this case, I don' think we need it. `embulk-input-config` should receive everything in the configuration.